pyproject.toml

```diff
@@ -0,0 +1,217 @@
+[tool.poetry]
+name = "openscm-units"
+version = "0.5.2"
+description = "Handling of units related to simple climate modelling."
+authors = [
+    "Zebedee Nicholls <zebedee.nicholls@climate-energy-college.org>",
+    "Jared Lewis <jared.lewis@climate-resource.com>",
+    "Robert Gieseke <rob.g@web.de>",
+    "Sven Willner <sven.willer@gmail.com>",
+]
+readme = "README.md"
+packages = [{include = "openscm_units", from = "src"}]
+
+[tool.poetry.dependencies]
+python = "^3.9"
+pint = "^0.22"
```

Comment:
I'd call that out in the changelog as we didn't previously have a minimum version for Pint